scripts/utils/sign_and_send.js

```diff
@@ -45,10 +45,10 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
       ADDRESS_0,
       nonce
     )
-    const lightWallet = await createLightwallet()
-    const account = lightWallet.accounts[0]
-    console.log(`Signing and posting multi-send transaction request from proposer account ${account}`)
-    const sigs = signTransaction(lightWallet, [account], transactionHash)
+    const privateKey = process.env.PK
+    const account = web3.eth.accounts.privateKeyToAccount(privateKey)
```

Comment:
This method is expecting a 0x prefixed string but the PK is exported without the 0x in truffle (https://web3js.readthedocs.io/en/v1.2.0/web3-eth-accounts.html#privatekeytoaccount)